tests/vmi_configuration_test.go

```diff
@@ -134,8 +133,9 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 	Context("when requesting virtio-transitional models", func() {
 		It("[test_id:6957]should start and run the guest", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
+			vmi := libvmi.NewCirros(
+				libvmi.WithRng(),
+			)
```

Comment:
nit: Could be oneline.

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/648daf3e11c01a0e5f69a3fc2dc4b986bdccb817..2595b29b5eb1b6d57354e2249ccd998a4b299e80)